mentors/javascript/c.json

```diff
@@ -38,5 +38,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I love learning to the point I got degrees in how/why people think the way they do and how to teach them. Then I learned how to program and have been trying to teach others as often as I can. Hope I can help."
+  },
+    {
```

Comment:
Remove two indents on line 42 and then this will be fine for merging.

Reply:
Done. Thanks